cmd/minishift/cmd/start.go

```diff
@@ -486,7 +486,7 @@ func determineIsoUrl(iso string) string {
 func initStartFlags() *flag.FlagSet {
 	startFlagSet := flag.NewFlagSet(commandName, flag.ContinueOnError)
 
-	startFlagSet.String(configCmd.ISOUrl.Name, minishiftConstants.B2dIsoAlias, "Location of the minishift ISO. Can be an URL, file URI or one of the following short names: [b2d centos].")
+	startFlagSet.String(configCmd.ISOUrl.Name, minishiftConstants.B2dIsoAlias, "Location of the minishift ISO. Can be an URL, file URI or one of the following short names: [b2d centos minikube].")
```

Comment:
👍

Reply:
-1

Reply:
Although the provisioner is not experimentally enabled, at least this string could be behind the toggle.

Reply:
... and the shorturl functionality that will likely follow this.

Reply:
@gbraad Even if we only show the usage of minikube short name when experimental is enabled it will still work if a user explicitly mentions it `minishift start --iso-url minikube`. Do you mean I should put a check to stop the execution if experimental features are not enabled and then minikube ISO is used.